packages/itmat-interface/src/graphql/resolvers/permissionResolvers.ts

```diff
@@ -8,18 +8,15 @@ import { errorCodes } from '../errors';
 import { IGenericResponse, makeGenericReponse } from '../responses';
 import { task_required_permissions, permissions } from 'itmat-commons';
 
-
 export const permissionResolvers = {
-    Query: {
```

Comment:
You have to add this back because when merging in resolvers/index.ts it will throw error..